golang/sample/gen_mobilenet_lib.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import tvm
+from tvm import relay
+import tvm.relay.testing
+
+######################################################################
+# Load Neural Network in Relay
+####################################################
+
+mod, params = relay.testing.mobilenet.get_workload(batch_size=1)
```

Comment:
Thanks for enhancing the as an end to end example.

How about using an official model than a workload ? 
i.e. Download mobilenet from official tensorflow, import and build. You may refer to tensorflow frontend test cases to download & import an official tensorflow model.

Reply:
Yes, I use the official model and TFLite frontend for the example, instead.